api/types/src/transaction.rs

```diff
@@ -607,8 +607,6 @@ pub enum GenesisPayload {
 pub enum TransactionPayload {
     EntryFunctionPayload(EntryFunctionPayload),
     ScriptPayload(ScriptPayload),
-    // Deprecated. Will be removed in the future.
-    ModuleBundlePayload(ModuleBundlePayload),
     MultisigPayload(MultisigPayload),
 }
```

Comment:
This probably shouldn't work.

Removing this, changes Multisig's enum value from a number perspective.

You'll have to leave it here, and change it to deprecated or something similar.

When BCS encoded, it will encode it as 0-3 for the type:

Reply:
Can we change the number everywhere else as well? Or it stops being compatible and this is the problem, right?

Reply:
we can't remove the variant but we can erase the inner thing from it, so it can become `DeprecatedModuleBundlePayload`,